prow/autobump.sh

```diff
@@ -73,21 +75,23 @@ extract-commit() {
   echo ${c##*-}
 }
 
-old_version=$(git show HEAD:prow/cluster/plank_deployment.yaml | extract-version)
-version=$(cat cluster/plank_deployment.yaml | extract-version)
-comparison=$(extract-commit "${old_version}")...$(extract-commit "${version}")
-echo -e "Pushing $(color-version ${version}) to ${user}:autobump..." >&2
+oncall=$(python2.7 -c 'import urllib2;import json;print json.loads(urllib2.urlopen("https://storage.googleapis.com/kubernetes-jenkins/oncall.json").read())["Oncall"]["testinfra"]')
```

Comment:
I dunno... between this and the new body (`\6`!) I suspect the complexity level has entered python or go territory.

Reply:
This is probably true.